app/controllers/donations_controller.rb

```diff
@@ -34,7 +34,11 @@ def index
 
     respond_to do |format|
       format.html
-      format.csv { send_data Donation.generate_csv(@donations), filename: "Donations-#{Time.zone.today}.csv" }
+      # TODO: can we do something fancy here to eagerly load the line
+      # items? (or maybe that's already been done for us?)
```

Comment:
You can move that eager loading into the service object :). You'd just need to use `includes` from what I recall.